tutornotes/templates/notes/build/notes/Dockerfile

```diff
@@ -5,9 +5,8 @@ RUN apt update && \
   apt upgrade -y && \
   # python 3.8
   apt install -y language-pack-en git python3 python3-pip libmysqlclient-dev
-RUN ln -s /usr/bin/python3 /usr/bin/python \
-    && ln -s /usr/bin/pip3 /usr/bin/pip
 
+RUN ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
I wonder why the `pip` symlink is no longer necessary... Maybe we can also get rid of the `python` symlink?

Reply:
I tried @regisb  but I got some errors

Reply:
Yes, I observed the same thing as you I think: https://github.com/overhangio/tutor-notes/pull/4